wrangler.toml

```diff
@@ -1,9 +1,9 @@
 name = "vhlink"
 type = "javascript"
-account_id = "39a76d640a29996bffbd58c36a537c2b"
+account_id = "670ade1150d4d3dc4b3eb2e9d48ea7ca"
```

Comment:
just substitute them back or make a new branch so it doesn't mess up your own instance I guess, we can't edit it before merging without editing your fork too